app/src/main/java/org/greenstand/android/TreeTracker/theme/CustomTheme.kt

```diff
@@ -0,0 +1,107 @@
+package org.greenstand.android.TreeTracker.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.greenstand.android.TreeTracker.R
+import org.greenstand.android.TreeTracker.view.AppColors
+
+@Immutable
+data class CustomColors(
+    val white: Color,
+    val black: Color,
+    val green: Color,
+    val orange: Color,
+)
+
+@Immutable
+data class CustomTypography(
+    val small: TextStyle,
+    val regular: TextStyle,
+    val medium: TextStyle,
+    val large: TextStyle
+)
+
+@Immutable
+data class CustomElevation(
+    val default: Dp,
+    val pressed: Dp
+)
+
+val LocalCustomColors = staticCompositionLocalOf {
+    CustomColors(
+        white = Color.Unspecified,
+        black = Color.Unspecified,
+        green = Color.Unspecified,
+        orange = Color.Unspecified
+    )
```

Comment:
Great suggestion. We can have the orange as uploadText and the green to primaryText. Is that okay?

Reply:
The colors are used for more than just text, for instance orange is used for text and button/progress bar. I'd keep the text color names separate from features. But we can name basic colors after features since we have color coded features.

Reply:
The colours here are just for text, the Colors.kt file containing all colours is still very much available and in use.

Reply:
I could refactor and have all the values of text colors fetched from the Colors.kt file so it would be kind of subset of Colors.kt

Reply:
If they are just for text, then please rename it to textColors and CustomTextColors